<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Raj" failures="1" tests="3" name="ListenerTest" time="0.052" errors="0" timestamp="2023-04-16T11:54:53 IST">
  <testcase classname="Listeners.SampleTestCaseListeners" name="equal2" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [5] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [5] but found [10]
at Listeners.SampleTestCaseListeners.equal2(SampleTestCaseListeners.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- equal2 -->
  <testcase classname="Listeners.SampleTestCaseListeners" name="equal3" time="0.0">
    <skipped/>
  </testcase> <!-- equal3 -->
  <testcase classname="Listeners.SampleTestCaseListeners" name="equal1" time="0.002"/>
</testsuite> <!-- ListenerTest -->
